app/uk/gov/hmrc/residencenilratebandcalculator/views/part_of_estate_passing_to_direct_descendants.scala.html

```diff
@@ -32,7 +33,7 @@
 
     @helpers.form(action = PartOfEstatePassingToDirectDescendantsController.onSubmit(), 'autoComplete -> "off") {
```

Comment:
The section tag above this needs a form-group class (either on the section or by wrapping the section in a div, I prefer the latter)

Reply:
This is true of any non p tag element sitting above a radio group